release.nix

```diff
@@ -61,7 +59,7 @@ let
     in
       buildFromConfig configuration (config: config.system.build.run-test);
 
-  release = import <nixpkgs/pkgs/top-level/release-lib.nix> {
+  release = import (nixpkgs + "/pkgs/top-level/release-lib.nix") {
```

Comment:
Out of curiosity, how are you accessing this from a flake/what are you using it for?

Reply:
I am trying to get hold of an options.json for https://search.nüschtos.de/ in https://github.com/NuschtOS/search.nuschtos.de/pull/22/files#diff-206b9ce276ab5971a2489d75eb1b12999d4bf3843b7988cbe8d687cfde61dea0R98 but because of the hatdcoded x86-64_linux platform as noted above, I ended up copying the important bits from this file.